README.md

```diff
@@ -1,57 +1,145 @@
-# cumulus-sled-java
+# cumulus-message-adapter-java
 
 [![CircleCI](https://circleci.com/gh/cumulus-nasa/cumulus-message-adapter-java.svg?style=svg)](https://circleci.com/gh/cumulus-nasa/cumulus-message-adapter-java)
 
-Java handler for the Cumulus Message Adapter. 
+## What is Cumulus?
 
-## Prerequisites
+Cumulus is a cloud-based data ingest, archive, distribution and management
+prototype for NASA's future Earth science data streams.
 
-  - Maven
+Read the [Cumulus Documentation](https://cumulus-nasa.github.io/)
 
-## Building and Packaging
+## What is the Cumulus Message Adapter?
 
-To build the uber-jar, run
+The Cumulus Message Adapter is a library that adapts incoming messages in the
+Cumulus protocol to a format more easily consumable by Cumulus tasks, invokes
+the tasks, and then adapts their response back to the Cumulus message protocol
+to be sent to the next task.
+
+## Installation
+
+With Maven installed, to build the uber-jar, run
```

Comment:
maybe Maven install instruction because they might not have it?

Reply:
Updated to include a link to install maven. Thank you!